src/main/java/seedu/partyplanet/model/person/Email.java

```diff
@@ -13,22 +13,16 @@
     public static final Email EMPTY_EMAIL = new Email();
 
     private static final String SPECIAL_CHARACTERS = "!#$%&'*+/=?`{|}~^.-";
-    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format local-part@domain "
+    public static final String MESSAGE_CONSTRAINTS = "Emails should be of the format USER@DOMAIN "
             + "and adhere to the following constraints:\n"
-            + "1. The local-part should only contain alphanumeric characters and these special characters, excluding "
+            + "1. USER can only contain alphanumerics and any of these special characters, excluding "
```

Comment:
Note that `_` is part of `\w` thus allowed too

Reply:
Good catch thanks! This is a bug inherited from AB3.